File: exmplos/coesao_desacoplamento/src/filho.py
from src.pai import Pai

class Filho(Pai):

    lista = []

    def cont_0_9(self, a):
        self.lista.append(a)
        for e in range(len(self.lista), 10, 1):
            return self.cont_0_9(len(self.lista))

File: exmplos/java/coesao_desacoplamento/oo/Filho.java
package oo;

public class Filho extends Pai {

    public Filho(){
        super();
    }

    public Filho(String cpf, String cnpj,
               String data_nescimento,
               String nome,
               String sobrenome){
        super.setCpf(cpf);
        super.setCnpj(cnpj);
        super.setData_nascimento(data_nescimento);
        super.setNome(nome);
        super.setSobrenome(sobrenome);
    }

    int[] lista = new int[10];

    public int cont_0_9(int a){
        return super.cont_0_9(a);
    }

}